components/UserTypeForm/UserTypeForm.tsx

```diff
@@ -13,9 +13,10 @@ const UserTypeForm = () => {
   const [userType, setUserType] = useState<number | null>()
   const dispatch = useDispatch()
   const { data } = useUserTypeForm()
-  const userTypeOptions = data?.type.choices || []
+  const emptyOption = { value: 0, display_name: '' }
+  const userTypeOptions = [emptyOption, ...(data?.type.choices || [])]
```

Comment:
can you explain why is this needed? i fixed the issue with the dropdown if that was your intention. the problem was that `Dropdown` used the label as defaultValue..

Reply:
this was my fix for the issue with dropdown. I will close this pr